eks/variables.tf

```diff
@@ -135,13 +145,13 @@ variable "max_capacity" {
 
 variable "min_capacity" {
   type        = number
-  default     = 5
+  default     = 6
   description = "The minimum number of worker nodes in the cluster"
 }
 
 variable "desired_capacity" {
   type        = number
-  default     = 5
+  default     = 6
```

Comment:
Let's limit this PR to the security improvements here too

Reply:
You're absolutely right, removed